yabt/builders/golang.py

```diff
@@ -116,71 +133,88 @@ def rm_all_but_go_mod(workspace_dir):
             rmtree(filepath)
 
 
-def go_builder_internal(build_context, target, command):
-    """Build or test a Go binary executable.
-    command is either build or test
+def generate_user_go_mod(build_context, target, build_cmd_env, go_module,
+                         user_mod_path):
+    user_mod_dir = dirname(user_mod_path)
+    user_mod_file = basename(user_mod_path)
+    build_context.run_in_buildenv(
```

Comment:
Isn't this command override the use file if it's there?

Can you add a comment explaing what's going on here?